### PR TITLE
docs(pipelinetemplate): Add note that pipeline templates have to be enabled

### PR DESCRIPTION
**Pull request by estebangarcia**
Add some documentation on enabling pipeline templates.

### CODE REVIEW

**Review by dorbin — COMMENTED** [review]
@estebangarcia 
Thanks for this!

I think because there's so little to do&mdash;two `hal` commands, we might not need this in /other_config/. What might make sense would be to put the new doc in /guides/user/pipeline/pipeline_templates/, in between Overview, and Managing Pipelines.

WDYT? Either way, whether in /setup/ or in /guides/, you'll want to add a corresponding entry to `_data/navigation.yml`.

**Comment by estebangarcia** [comment]
@dorbin I think it makes sense and I'll make it more visible.

You mean in between Overview and Create a Pipeline Template right?

**Comment by dorbin** [comment]
Yes, @estebangarcia that's what I meant. Wasn't paying close attention to what I was writing. :1st_place_medal:

**Comment by estebangarcia** [comment]
@dorbin I made the changes.

File: guides/user/pipeline/pipeline-templates/index.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ Spinnaker itself.
 > **Note**: You can use `spin` CLI to manage pipelines and pipeline templates,
 > but first you need to [install it](/guides/spin/cli/).
 
+> **Note**: You first need to [enable the feature](/guides/user/pipeline/pipeline-templates/enable/) if you haven't done it yet
```
Comment (dorbin) [inline guides/user/pipeline/pipeline-templates/index.md]:
```suggestion
> **Note**: You first need to [enable the feature](/guides/user/pipeline/pipeline-templates/enable/) if you haven't done it yet.
```

**Review by dorbin — APPROVED** [review]

**Pull request merged**